configs/openmrs_config/concepts/ethnicities.csv

```diff
@@ -28,5 +28,4 @@ bac9af94-07b3-44f2-b678-5f02ad5959e4,,,Robel,រ៉ូបែល,Robel,រ៉ូ
 8d57672b-bc62-49dd-a417-ef62b8522275,,,Poang,ប៉ូអាង?,Poang,ប៉ូអាង?,,,Misc,Text,,,,
 769d258d-c6f8-477e-a5ed-10ff07c80fe3,,,Roong,រ៉ូអ៊ុង,Roong,រ៉ូអ៊ុង,,,Misc,Text,,,,
 57fe19cb-c83e-4778-99f6-a20d55b40191,,,Other Indigenous,ជនជាតិដើមភាគតិចផ្សេងទៀត,Other Indigenous,ជនជាតិដើមភាគតិចផ្សេងទៀត,,,Misc,Text,,,,
-5622AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,Other,ផ្សេងទៀត,Other,ផ្សេងទៀត,,,Misc,Text,,,,
```

Comment:
@kdaud, did you update the Uuid for all occurences of this in the forms to match the new Uuid for `Other`

Reply:
The uuid hasn't changed but rather refactoring `Other` concept from being duplicated.